docker.dev.yml

```diff
@@ -1,10 +1,22 @@
 version: "3"
 
 services:
-  app:
-    container_name: scribble-ocr
+  train:
+    container_name: scribble-ocr-train
     build:
       context: .
+      target: tesseractbuild
+      dockerfile: Dockerfile_tesseract_training
+    ports:
```

Comment:
There isn't any reason to set ports on the training image since it doesn't expose an API at all.